x11-libs/libxft/libxft-2.3.2.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="FreeType-based font drawing library for X"
+DESCRIPTION="Xft provides a client-side font API for X applications, \
+making the FreeType font rasterizer available to X clients. Fontconfig \
+is used for font specification resolution. Where available, the RENDER \
+extension handles glyph drawing; otherwise, the core X protocol is used."
+HOMEPAGE="https://www.x.org/"
+COPYRIGHT="2000-2003 Keith Packard"
+LICENSE="libXft"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXft-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f5a3c824761df351ca91827ac221090943ef28b248573486050de89f4bfcdc4c"
+SOURCE_DIR="libXft-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+libVersion="4.11.0"
```

Comment:
```diff
-libVersion="4.11.0"
+libVersion="$portVersion"
```
That's not very common, but the version of the shared lib is the same as that of the package.

Reply:
Fixed.